app/test_main.py

```diff
@@ -1,3 +1,24 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    ("cat_age", "dog_age", "expected"),
+    [
+        (0, 0, [0, 0]),
```

Comment:
This test case `(0, 0, [0, 0])` is a duplicate and should be removed to avoid redundancy in the test suite.